microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -323,7 +327,7 @@ See [Types](https://developer.chrome.com/docs/extensions/reference/sidePanel/#ty
 <!-- ====================================================================== -->
 > [!NOTE]
 > Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
-> The original page is found [here](https://developer.chrome.com/docs/extensions/reference/sidePanel/).
+> The original page is found [here](https://developer.chrome.com/docs/extensions/reference/api/sidePanel).
```

Comment:
pls take a look at updated upstream article:
https://developer.chrome.com/docs/extensions/reference/api/sidePanel
@bhuvanapriyap 

rendered article in PR:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328

Reply:
Seems good to me, the old URL now redirects to the new one.